MoreLinq.Test/OrderedMergeTest.cs

```diff
@@ -0,0 +1,398 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+
+namespace MoreLinq.Test
+{
+    [TestFixture]
+    class OrderedMergeTest
+    {
+        [Test]
+        public void ShouldBeLazy()
+        {
+            var first = new BreakingSequence<object>();
+            var second = new BreakingSequence<object>();
+            Assert.DoesNotThrow(() => MoreEnumerable.OrderedMerge(first, second, id => id, id => id, id => id, id => id, (f, _) => f, null));
+        }
+
+        [Test]
+        public void ShouldDisposeEnumerators()
+        {
+            using (var first = TestingSequence.Of(new int[] { }))
```

Comment:
`TestingSequence.Of(new int[] { })` can be reduced to simply `TestingSequence.Of<int>()`